src/commands/android/utils/adbWirelessConnect.ts

```diff
@@ -0,0 +1,73 @@
+import {Platform} from '../interfaces';
+import {execBinarySync} from './sdk';
+import Logger from '../../../logger';
+import inquirer from 'inquirer';
+import colors from 'ansi-colors';
+
+export async function connectAdbWirelessly(adbLocation: string, platform: Platform): Promise<boolean> {
+  try {
+    const deviceList = execBinarySync(adbLocation, 'adb', platform, 'devices -l');
+    if (deviceList !== null) {
+      if (!deviceList.toString().includes('transport_id')) {
+        Logger.log(colors.red('No devices found. Please connect a device and try again.'));
+
+        return false;
+      }
+    }
+
+    Logger.log(`${colors.bold('Follow the below steps to connect to your device wirelessly:\n')}
+  1. Enable USB debugging on your device.
+  2. Connect your device to your computer using a USB cable.
```

Comment:
Like I said before, we shouldn't ask or expect users to connect their device using a USB cable. You don't need connect using a USB cable to do the pairing, it can be done wirelessly as well.